client/public/elements/components/a-z.js

```diff
@@ -73,13 +73,14 @@ export class RpAZ extends LitElement {
       }
     }
 
-
-
     if( this.baseHref ) {
       let disabled = this.disabledLettersFmt.includes(letter);
       let href = "#";
       if( !disabled ) {
-        href = `/${this.baseHref}?az=${letter.toLowerCase()}`;
+        let azlink=new URL(this.baseHref,window.location.href);
```

Comment:
@sbagg This is probably safer than the commented out solution, but I'm not sure what the best method for fetching a URL really is, and whether we should make them relative for some reason.

Reply:
@jrmerz, I added two solutions here.  I was just concerned that using the builtin URL() was a problem because you have to specify the base, and also maybe it'd have to be polyfilled for edge?